### PR TITLE
[MIG] document_page_tags

### PR DESCRIPTION
**Pull request by hbrunn**
actual migration in https://github.com/OCA/knowledge/commit/9a57c7eb2cde0c10ccae5cfadf9ec5f1f24044f9

### CODE REVIEW

File: document_page_tags/models/document_page_tag.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    name = fields.Char(required=True, translate=True)
+
+    _sql_constraints = [
+        ('unique_name', 'unique(name)', 'Tags must me unique'),
+    ]
+
+    @api.model
+    def create(self, vals):
+        """Be nice when trying to create duplicates"""
+        existing = self.search([('name', '=ilike', vals['name'])])
```
Comment (daramousk) [inline document_page_tags/models/document_page_tag.py]: Maybe put a limit here?

File: document_page_tags/tests/__init__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (daramousk) [inline document_page_tags/tests/__init__.py]: You might want to change the old style header.

**Comment by hbrunn** [comment]
@daramousk thanks, done

**Review by daramousk — APPROVED** [review]
LGTM

**Comment by ivantodorovich** [comment]
Shouldn't this be an improvement on document_page, instead of a separate module?

**Review by max3903 — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@max3903 why don't you wait for the answer about https://github.com/OCA/knowledge/pull/180#issuecomment-421491762. The name is also against guidelines as it contains a plural.

**Comment by max3903** [comment]
@pedrobaeza Sorry I didn't pay attention and see the question.

**Pull request merged**